lib/mvos/model/entity/user_entity.dart

```diff
@@ -1,22 +1,44 @@
 import 'package:flutter/foundation.dart';
 
-class UserE {
+class FirebaseUserE {
+  final String name;
+  final String surname;
+  final String email;
+  final String password;
```

Comment:
nisi jos maknuo password :D

Reply:
password handla firebase auth. nikad ne spremas password in plain text. samo zoves funkcije s njim

Reply:
ok